packages/suite/src/components/onboarding/OnboardingStepBox/index.tsx

```diff
@@ -21,14 +21,12 @@ const InnerActions = styled.div`
 
 const ConfirmWrapper = styled.div`
     margin-bottom: 20px;
```

Comment:
Not sure if it this one. But skip window in onboarding is not visible
![Screenshot 2022-05-18 at 9 37 24](https://user-images.githubusercontent.com/33235762/168983948-7da2484d-3e53-4fcb-8336-3d7b4ed3626d.png)

Reply:
Fixed.